rust/lance-file/src/format.rs

```diff
@@ -15,5 +15,5 @@ pub mod metadata;
 
 /// These version/magic values are written at the end of Lance files (e.g. versions/1.version)
 pub const MAJOR_VERSION: i16 = 0;
-pub const MINOR_VERSION: i16 = 1;
+pub const MINOR_VERSION: i16 = 2;
```

Comment:
What kind of error do we get if an old version of lance tries to read a new index file?

Reply:
for the current impl, it will report a proto parsing error, for the old version of lance, the new index file is something like a corrupted index file